pkgs/development/python-modules/pyspcwebgw/default.nix

```diff
@@ -25,9 +23,18 @@ buildPythonPackage rec {
     hash = "sha256-gdIrbr25GXaX26B1f7u0NKbqqnAC2tmMFZspzW6I4HI=";
   };
 
-  nativeBuildInputs = [ poetry-core ];
+  patches = [
+    # https://github.com/pyspcwebgw/pyspcwebgw/pull/27
+    (fetchpatch {
+      name = "replace-async-timeout-with-asyncio.timeout.patch";
+      url = "https://github.com/pyspcwebgw/pyspcwebgw/commit/22cacc8db53cf2a244c30c0e62a0dad90fbcb00b.patch";
+      hash = "sha256-Og0imZts49jwjbz7Yp41UIzwU/lVjKVc/Tp4+vNz32U=";
+    })
+  ];
```

Comment:
Do you think it would be better to perhaps [vendor the patch](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#vendoring-patches)? upstream looks a bit dead so I don't think this will be merged any time soon.